src/main/scala/de/htwg/se/sudoku/model/fileIoComponent/fileIoMicroImpl/FileIO.scala

```diff
@@ -19,7 +19,7 @@ import scala.concurrent.{Await, ExecutionContextExecutor, Future}
 import scala.util.{Success, Try}
 
 class FileIO @Inject()(@Named("FileHost") host: String, @Named("FilePort") port: Int) extends FileIOInterface {
-  implicit val system: ActorSystem = ActorSystem()
+  implicit val system: ActorSystem = ActorSystem("FileIoHttpServerSystem")
```

Comment:
shouldn't this be `FileIoServerSystem`?

Reply:
I think it's okay, the class is called `FileIoHttpServer` and the variables also...

Is your idea to change everything from `FileIoHttpServer` to `FileIoServer`?

Reply:
The class is called `FileIO`, am I missing something here? 🧐

Reply:
I used before in the `FileIoHttpServer` and the `FileIO` the same system and that makes completely no sense 😂 👍 

Including https://github.com/bb30/de.htwg.se.SudokuInScala/pull/11/commits/dc554867ff7d7608ae8110aad6db05bd343780fe it should be better...